bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml

```
@@ -37,6 +37,9 @@ metadata:
                 "name": "activemq"
               }
             }
+          },
+          "status": {
+            "conditions": null
```

is this null value required?

---

@brusdev  The null values are auto generated. They don't represent any real conditions. but the `status.conditions` has to be there in order to pass the test.

---

I guess you need an non-empty status to fix the scorecard test warning. If this is the case you could use an empty array for conditions to avoid the null value in the examples, i.e.
```suggestion
            "conditions": []
```

---

sure, i will modify with empty array here.